tensorbay/client/dataset.py

```diff
@@ -512,56 +541,64 @@ def _list_diff_segments(
 
         Examples:
-            >>> self._list_diff_segments(source="b382450220a64ca9b514dcef27c82d9a", target=1)
-            {
-                "segments": [
-                    {
-                        "name": "Segment1",
-                        "action": "add",
-                        "data": {
-                            "stats": {
-                                "total": 3,
-                                "additions": 3,
-                                "deletions": 0,
-                                "modifications": 0
-                            },
-                        },
-                        "sensors": {
-                            "action": "modify"
-                        }
-                    },
-                    {
-                        "name": "Segment2",
-                        "action": "add",
-                        "data": {
-                            "stats": {
-                                "total": 3,
-                                "additions": 3,
-                                "deletions": 0,
-                               "modifications": 0
-                            },
-                        },
-                        "sensors": {
-                            "action": "modify"
-                        }
-                    },
-                ]
-                ...
-                "offset": 0,
-                "recordSize": 2,
-                "totalCount": 2
-            }
+            >>> self.get_diff(source="b382450220a64ca9b514dcef27c82d9a", target=1)
 
         Returns:
-            The PagingList of diffs of each segment.
+            The brief diff between two versions.
 
         """
         basehead = self._get_basehead(source, target)
 
-        return PagingList(
-            lambda offset, limit: self._generate_diff_segments(basehead, offset, limit), 128
+        dataset_diff = DatasetDiff(self.name)
+        segment_diffs = PagingList(
+            lambda offset, limit: self._generate_segment_diffs(basehead, offset, limit), 128
+        )
+
+        dataset_diff._segments = (  # pylint: disable=protected-access
+            segment_diffs  # type: ignore[assignment]
+        )
+
+        return dataset_diff
+
+    def get_segment_diff(
+        self,
+        segment_name: str,
+        *,
+        source: Optional[Union[str, int]] = None,
+        target: Optional[Union[str, int]] = None,
+    ) -> SegmentDiff:
+        """Get diff of the segment between two versions.
+
+        Arguments:
+            segment_name: The name of the segment.
+            source: Source version identification. Type int for draft number, type str for revision.
+                If is not given, use the current version.
+            target: Target version identification. Type int for draft number, type str for revision.
+                If is not given, use the parent commit id.
+
+        Examples:
+            >>> self.get_segment_diff(segment_name="Segment1", \
```

Comment:
Is the `segment_name` necessary? Is this the recommended way to call this method?